profanities.go

```diff
@@ -3,68 +3,397 @@ package goaway
-	"felching",
+	"fingerfuck",
+	"fingerfucked",
+	"fingerfucker",
+	"fingerfuckers",
+	"fingerfucking",
+	"fingerfucks",
+	"fistfuck",
+	"fistfucked",
+	"fistfucker",
+	"fistfuckers",
+	"fistfucking",
+	"fistfuckings",
+	"fistfucks",
+	"flange",
+	"fook",
+	"fooker",
 	"fuck",
+	"fucka",
+	"fucked",
+	"fucker",
+	"fuckers",
+	"fuckhead",
+	"fuckheads",
+	"fuckin",
+	"fucking",
+	"fuckings",
+	"fuckingshitmotherfucker",
+	"fuckme",
+	"fucks",
+	"fuckwhit",
+	"fuckwit",
 	"fudgepacker",
-	"flange",
+	"fuk",
+	"fuker",
+	"fukker",
+	"fukkin",
+	"fuks",
+	"fukwhit",
+	"fukwit",
+	"fux",
+	"fuxor",
+	"gangbang",
+	"gangbanged",
+	"gangbangs",
+	"gaylord",
+	"gaysex",
+	"goatse",
+	"God",
+	"goddam",
+	"goddamned",
+	"goddamn",
+	"goddamned",
+	"hardcoresex",
+	"hell",
+	"heshe",
+	"hoar",
+	"hoare",
+	"hoer",
+	"homo",
+	"hore",
+	"horniest",
 	"horny",
+	"hotsex",
 	"incest",
+	"jackoff",
+	"jackoff",
+	"jap",
 	"jerk",
+	"jerkoff",
+	"jism",
+	"jiz",
+	"jizm",
 	"jizz",
+	"kawk",
+	"knobead",
+	"knobed",
+	"knobend",
+	"knobhead",
+	"knobjocky",
+	"knobjokey",
+	"kock",
+	"kondum",
+	"kondums",
+	"kum",
+	"kummer",
+	"kumming",
+	"kums",
+	"kunilingus",
 	"labia",
+	"masochist",
+	"masterbate",
+	"masterbat",
+	"masterbate",
+	"masterbation",
+	"masterbations",
+	"masturbate",
+	"materbation",
+	"mofo",
+	"mothafuck",
+	"mothafucka",
+	"mothafuckas",
+	"mothafuckaz",
+	"mothafucked",
+	"mothafucker",
+	"mothafuckers",
+	"mothafuckin",
+	"mothafucking",
+	"mothafuckings",
+	"mothafucks",
+	"motherfuck",
+	"motherfucked",
+	"motherfucker",
+	"motherfuckers",
+	"motherfuckin",
+	"motherfucking",
+	"motherfuckings",
+	"motherfuckka",
+	"motherfucks",
 	"muff",
-	"naked",
-	"nigger",
+	"mutha",
+	"muthafecker",
+	"muthafuckker",
+	"muther",
+	"mutherfucker",
+	"nazi",
 	"nigga",
-	"nude",
+	"niggah",
+	"niggas",
+	"niggaz",
+	"nigger",
+	"niggers",
+	"nob",
+	"nobjokey",
+	"nobhead",
+	"nobjocky",
+	"nobjokey",
+	"numbnuts",
+	"nutsack",
+	"orgasim",
+	"orgasims",
+	"orgasm",
+	"orgasms",
+	"pawn",
+	"pecker",
 	"penis",
+	"penisfucker",
+	"phonesex",
+	"phuck",
+	"phuk",
+	"phuked",
+	"phuking",
+	"phukked",
+	"phukking",
+	"phuks",
+	"phuq",
+	"pigfucker",
+	"pimpis",
 	"piss",
+	"pissed",
+	"pisser",
+	"pissers",
+	"pisses",
+	"pissflaps",
+	"pissin",
+	"pissing",
+	"pissoff",
 	"poop",
 	"porn",
+	"porno",
+	"pornos",
 	"prick",
+	"pricks",
+	"pron",
 	"pube",
+	"pusse",
+	"pussi",
+	"pussies",
 	"pussy",
+	"pussys",
 	"queer",
+	"rectum",
+	"retard",
 	"rimjob",
+	"rimjaw",
+	"rimming",
+	"schlong",
+	"screwing",
+	"scroat",
+	"scrote",
 	"scrotum",
-	"sex",
+	"semen",
+	"shag",
+	"shagger",
+	"shaggin",
+	"shagging",
+	"shemale",
 	"shit",
+	"shitdick",
+	"shite",
+	"shited",
+	"shitey",
+	"shitfuck",
+	"shitfull",
+	"shithead",
+	"shiting",
+	"shitings",
+	"shits",
+	"shitted",
+	"shitter",
+	"shitters",
+	"shitting",
+	"shittings",
+	"shitty",
+	"skank",
 	"slut",
+	"sluts",
+	"smegma",
+	"smut",
+	"snatch",
+	"sonofabitch",
+	"spac",
```

Comment:
- `snatch` is more often not used as a profanity than it is
- `sonofabitch` is handled by `bitch`
- `spac` is a little too broad to be considered a profanity
```suggestion
```